### PR TITLE
EIP-4883 Composable SVG  NFT

### PR DESCRIPTION
**Pull request by abcoathup**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by abcoathup** [comment]
This EIP is currently a draft.

File: EIPS/eip-4888.md
Comment (MicahZoltu) [inline EIPS/eip-4888.md:11]:
```suggestion
requires: 721
```

File: EIPS/eip-4888.md
Comment (MicahZoltu) [inline EIPS/eip-4888.md:9]:
```suggestion
category: ERC
```

**Review by MicahZoltu — DISMISSED** [review]
At a minimum, a specification is required for a Draft EIP.  If you don't have a specification ready, I recommend closing this PR and waiting to open it until you do.

EIP number sniping and gaming is not allowed, and since this PR was open a mere 2 minutes after PR 4887 and doesn't contain any meaningful content, I'm overriding our normal numbering system in favor of a manual assignment of 4883 here.

**Comment by abcoathup** [comment]
> EIP number sniping and gaming is not allowed, and since this PR was open a mere 2 minutes after PR 4887 and doesn't contain any meaningful content, I'm overriding our normal numbering system in favor of a manual assignment of 4883 here.

I check the repo most days (as part of my job), saw that a meaningful number to me was next up and thought I had won the lottery so did a PR with a very early draft.  Definitely not number gaming (unlike say 4844), more of a lunge, I'll update the draft next week.

File: EIPS/eip-4888.md
Comment (abcoathup) [reply EIPS/eip-4888.md:6]: I was waiting for the number confirmation as can't edit Eth Magicians forum posts after a while.

Comment (MicahZoltu) [reply EIPS/eip-4888.md:6]: Ethereum Magicians no longer has a time limit on editing posts!  This is new as of this week, partially to address this exact problem.  😄

**Comment by SamWilsn** [comment]
Let me know when you open your discussion thread. I've seen at least [EIP-4841](https://github.com/ethereum/EIPs/pull/4841) by @Soohan-Park which is also proposing extensions for better SVG support. Might be a good idea to get an informal working group together!

**Comment by AlexPartyPanda** [comment]
> Let me know when you open your discussion thread

My discussion thread post to Ethereum-magicians is currently pending, waiting for their moderators to approve

**Comment by Bezzat** [comment]
Hello yes broo please help me 🙏

30 Mar 2022 Çar 09:24 tarihinde Alex Panda ***@***.***> şunu
yazdı:

**Comment by MicahZoltu** [comment]
> My discussion thread post to Ethereum-magicians is currently pending, waiting for their moderators to approve

@poojaranjan Hmm, is this something we can tune?  What are the conditions that trigger needing approval?

**Comment by poojaranjan** [comment]
@MicahZoltu It shouldn't be long waiting for a discussion link to be live. Checking with the FEM team to figure if it is an exceptional case. 

@AlexPartyPanda Can you please share the link here or maybe with me at [ECH Discord](https://discord.io/EthCatHerders).

**Comment by PradhumnaPancholi** [comment]
Thanks, @poojaranjan, for getting it live.

**Comment by abcoathup** [comment]
> As it appears, Discourse sometimes choose to hold a post, often it is a new users or even if they seem to be performing the task "too quickly".

Thanks @poojaranjan.  

@AlexPartyPanda had a new account on Eth magicians, so was limited to including two links in his post.  We tried to work around this and when we did post, pasted the whole text of the discussion in one go, so was picked up by Discourse as potential spam.

Something to be aware of for EIP discussions that new users are limited to two links.

File: EIPS/eip-4883.md
Comment (SamWilsn) [inline EIPS/eip-4883.md:49]: This should be included in the "Specification" section.

The Specification section says "what" is proposed by the EIP, and Rationale says "why" those particular changes were chosen.

File: EIPS/eip-4883.md
Comment (SamWilsn) [inline EIPS/eip-4883.md:57]: Would you mind just adding a quick: This EIP has no backwards compatibility concerns?

File: EIPS/eip-4883.md
Comment (SamWilsn) [inline EIPS/eip-4883.md:11]: Probably also needs 165.

File: EIPS/eip-4883.md
Comment (Pandapip1) [inline EIPS/eip-4883.md:66]:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE.md).
```

File: EIPS/eip-4883.md
Comment (Pandapip1) [reply EIPS/eip-4883.md:57]: If #5118 is merged this will be unneccesary.

**Comment by abcoathup** [comment]
Reopen to run new EIP linter

File: EIPS/eip-4883.md
Comment (Pandapip1) [inline EIPS/eip-4883.md:38]:
```suggestion

SVG elements can be string concatenated to compose an SVG.
```

File: EIPS/eip-4883.md
Comment (Pandapip1) [inline EIPS/eip-4883.md:71]:
```suggestion
## Security Considerations

- SVG uses a "painters model" of rendering. A rendered SVG body could be added and completely obscure the existing SVG NFT artwork.
- SVG is XML and can contain malicious content, and while it won't impact the contract, it could impact the use of the SVG.
```

**Comment by bitcoinbrisbane** [comment]
Similar idea to my https://www.modadao.io/ NFT project.   We create vector art from sampling music https://noiseless-orange.surge.sh/

**Comment by abcoathup** [comment]
Trigger CI checks

File: EIPS/eip-4883.md
Comment (Pandapip1) [inline EIPS/eip-4883.md:27]:
```suggestion
```
No external links.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
LGTM except for this.

File: EIPS/eip-4883.md
Comment (abcoathup) [reply EIPS/eip-4883.md]: Removed the link in the code.

**Review by Pandapip1 — APPROVED** [review]

**Comment by abcoathup** [comment]
Auto-Merge Bot fails even though @Pandapip1 has reviewed?
>File with name EIPS/eip-4883.md is new and new files must be reviewed
This PR requires review from one of [lightclient, axic, samwilsn, pandapip1]

**Comment by Pandapip1** [comment]
Close and re-open the PR. There's a fix in #2400 but it has yet to be merged.

**Pull request merged**
